javascript/ql/src/Security/CWE-834/TaintedLength.qhelp

```diff
@@ -7,32 +7,18 @@
     <p>
         Iterating the elements of an untrusted object using the 
         <code>.length</code> property can lead to a server looping 
-        indefinitely or crashing. Thereby creating a 
+        indefinitely or crashing. This looping or crashing creates a 
```

Comment:
By "crashing" i meant crashing the entire process (usually because out-of-mem). 

But i can see that it wasn't clear, so I'll focus on looping indefinitely.